oracle-linux-image-tools/README.md

```diff
@@ -33,6 +36,7 @@ The tool currently supports:
   `yum --enablerepo=ol7_developer install packer VirtualBox-6.0`
 1. For `OCI` or `OLVM` images, install the ` qemu-img` package:  
   `yum install qemu-img`
+1. For `Vagrant` box (VirtualBox provider), install [Vagrant](https://vagrantup.com/)
```

Comment:
Can we include HashiCorp in the name of the product, i.e. `[HashiCorp Vagrant]`? Also, highlight the fact that we ship Vagrant on yum.oracle.com so you can just enable the developer repo and `yum install vagrant`.

Reply:
Unless I am missing something, we don't ship Vagrant (we do ship Terraform)...

Reply:
My bad. We currently ship Packer and Terraform, not Vagrant. Though, I suspect we should. :)